autoemulate/core/compare.py

```diff
@@ -120,13 +127,25 @@ def __init__(
             it will show a progress bar during model comparison. It will set the
             logging level to "error" to avoid cluttering the output
             with debug/info logs.
+        tuning_metric: str | TorchMetrics
+            Metric to use for hyperparameter tuning. Can be a string shortcut
+            ("r2", "rmse", "mse", "mae") or a MetricConfig object. Defaults to "r2".
+        evaluation_metrics: list[str | TorchMetrics] | None
```

Comment:
It looks like we use the first metric in this list for selecting the best performing model, can we add this to the docs to make it clear this is the case (and log the choice as well)?